### ISSUE TITLE
[doc] Generate 2.7.1 doc set for Pulsar

### ISSUE THREAD

**Issue by Jennifer88huang** [streamnative/pulsar#1920]
Review https://github.com/apache/pulsar/wiki/Release-process
Try to generate doc set for 2.7.1 and refine the workflow related to doc set

**Comment by Jennifer88huang** [comment]
- While i'm trying to follow the [release workflow](https://github.com/apache/pulsar/wiki/Release-process#16-update-the-site) to generate a whole doc set for 2.7.1, nothing comes out.
- While I'm trying to do it for 2.6.3, "permission denied" error occurs.
The following is code snppit
```
Jennifers-MacBook-Pro:pulsar jenniferhuang$ git checkout -b 271
Switched to a new branch '271'
Jennifers-MacBook-Pro:pulsar jenniferhuang$ cd site2/website
Jennifers-MacBook-Pro:website jenniferhuang$ yarn install
yarn install v1.15.2
warning package.json: No license field
warning No license field
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.55s.
arnJennifers-MacBook-Pro:website jenniferhuang$ yarn run version 2.7.1
yarn run v1.15.2
warning package.json: No license field
$ cp docusaurus-version.js ./node_modules/.bin/ && docusaurus-version 2.7.1
Version 2.7.1 created!

✨  Done in 5.91s.
Jennifers-MacBook-Pro:website jenniferhuang$ git add versioned_docs/*
Jennifers-MacBook-Pro:website jenniferhuang$ git add versioned_sidebars/*
Jennifers-MacBook-Pro:website jenniferhuang$ yarn run version 2.7.1
yarn run v1.15.2
warning package.json: No license field
$ cp docusaurus-version.js ./node_modules/.bin/ && docusaurus-version 2.7.1
This version already exists!
Specify a new version to create that does not already exist.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Jennifers-MacBook-Pro:website jenniferhuang$ yarn run version 2.6.3
yarn run v1.15.2
warning package.json: No license field
$ cp docusaurus-version.js ./node_modules/.bin/ && docusaurus-version 2.6.3
Version 2.6.3 created!

✨  Done in 3.96s.
Jennifers-MacBook-Pro:website jenniferhuang$ yarn add versioned_docs/*
yarn add v1.15.2
warning package.json: No license field
warning No license field
[1/4] 🔍  Resolving packages...
error Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads ssh://git@github.com/versioned_docs/version-2.1.0-incubating.git
Directory: /Users/jenniferhuang/GitHub/pulsar/site2/website
Output:
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
Jennifers-MacBook-Pro:website jenniferhuang$ 

```

**Comment by Jennifer88huang** [comment]
fix it in apache/pulsar/pull/9026

**Comment by Jennifer88huang** [comment]
Generate the whole doc set with the following command
`yarn run version <release-version>`
